umputun/remark42
# Update frontend

I spent a few days investigating how we can reduce our frontend dependencies number but it's "modern frontend" and I tried but it not a significant difference.

* use absolute path resolving
* update deps to latest versions
* use latest tools for building bundles
* rewrite webpack config 
* use nomodule technique for loading modern bundle
* inject polyfills by babel usebuiltins
* update eslint rules
* rename all style files to CSS
* use postcss preset env for building styles
* proper typescript typing
* put all html files in the templates folder
* etc

- [x] Fix readme
- [x] Check if everything works as before



Bundle sizes compassion:
 |                                              | Old                        | Legacy          | Modern    |
 |---------------------------|------------------|--------------|-----------|
 | public/embed.js                  | 2.68 KB                 |  2.21 KB          | 2.21 KB    |
 | public/remark.js                  | 83.13 KB               |  72.87 KB       | 68.81 KB  |
 | public/last-comments.js    | 42.27 KB               |  37.72 KB       | 30.56 KB  |
 | public/deleteme.js              | 33.49 KB               |  17.58 KB       | 10.44 KB   |
 | public/counter.js                 | 1.04 KB                  |  606 B            | 606 B       |
 
 Old - current bundle sizes
 Legacy - new bundle sizes for old browsers
 Moders - new bundle sizes for modern browsers
